query/sql/sql_instance_postgresql_log_error_verbosity_database_flag_set_default_or_stricter.sql

```diff
@@ -0,0 +1,20 @@
+select 
+  -- Required Columns
+  self_link resource,
```

Comment:
Change query name to `sql_instance_postgresql_log_error_verbosity_database_flag_default_or_stricter.sql`